install_radar_deps.py

```diff
@@ -0,0 +1,408 @@
+#!/bin/python3
+
+"""
+Copyright SuperDARN Canada 2020
+Keith Kotyk
+
+Installation script for Borealis utilities.
+NOTE: This script has been tested on:
+    OpenSuSe 15.1
+    Ubuntu 19.10
+    Ubuntu 20.04
+"""
+import subprocess as sp
+import sys
+import os
+import multiprocessing as mp
+import argparse as ap
+
+def usage_msg():
+    """
+    Return the usage message for this process.
+
+    This is used if a -h flag or invalid arguments are provided.
+
+    :returns: the usage message
+    """
+
+    usage_message = """ install_radar_deps.py [-h] radar install_dir
+
+    This script will download and configure all dependencies for the
+    Borealis project. You must supply an installation directory.
+
+    Example usage:
+    python3 install_radar_deps.py sas /home/radar/borealis/
+
+    """
+    return usage_message
+
+def execute_cmd(cmd):
+    """
+    Execute a shell command and return the output
+
+    :param      cmd:  The command
+    :type       cmd:  string
+
+    :returns:   Decoded output of the command.
+    :rtype:     string
+    """
+    output = sp.check_output(cmd, shell=True)
+    return output.decode('utf-8')
+
+def get_distribution():
+    """
+    Gets the linux distribution type.
+
+    :returns:   The distribution name.
+    :rtype:     str
+    """
+
+    os_info = execute_cmd("cat /etc/os-release")
+
+    os_info = os_info.splitlines()
+    distro = os_info[0].strip("NAME=").strip('"')
+
+    return distro
+
+def install_packages():
+    """
+    Install the needed packages used by Borealis. Multiple options are listed for distributions that
+    use different names.
+    """
+
+    packages = ["wget",
```

Comment:
I tested with Ubuntu. The first errors I got were in installing some of these packages. Do we expect some of them to fail, based on the comment that multiple options are listed for distributions using different names? 

As someone who hasn't done this before this is pretty confusing. If something doesn't work I won't know which packages were expected to fail because of my distro and which weren't.